next-tavla/src/Admin/scenarios/CreateBoard/components/AddStops.tsx

```diff
@@ -0,0 +1,140 @@
+import { useStopPlaceSearch } from 'Admin/scenarios/Edit/hooks/useStopPlaceSearch'
+import { createBoardRequest } from '../utils/create'
+import { useRouter } from 'next/router'
+import { StopPlaceChip } from './StopPlaceChip'
+import { useToast } from '@entur/alert'
+import { TBoard } from 'types/settings'
+import { useCreateBoardDispatch } from '../utils/context'
+import { useQuaySearch } from 'Admin/scenarios/Edit/hooks/useQuaySearch'
+
+export function AddStops({
+    board,
+    popPage,
+}: {
+    board: TBoard
+    popPage: () => void
+}) {
+    const router = useRouter()
+    const { addToast } = useToast()
+    const { counties, selectedCounties, setSelectedCounties } =
+        useCountiesSearch()
+
+    const { stopPlaceItems, selectedStopPlace, setSelectedStopPlace } =
+        useStopPlaceSearch(selectedCounties.map((county) => county.value))
+
+    const { quays, selectedQuay, setSelectedQuay } = useQuaySearch(
+        selectedStopPlace?.value ?? '',
+    )
+    const dispatch = useCreateBoardDispatch()
+
+    const handleAddTiles = () => {
+        if (!selectedStopPlace?.value) {
+            addToast({
+                title: 'Ingen holdeplass er valgt',
+                content: 'Vennligst velg en holdeplass å legge til',
+                variant: 'info',
+            })
+            return
+        }
+        if (selectedQuay) {
+            dispatch({
+                type: 'addTile',
+                tile: {
+                    type: 'quay',
+                    placeId: selectedQuay.value,
+                    name:
+                        selectedStopPlace.label.split(',')[0] +
+                            ' ' +
+                            selectedQuay.label ?? 'Ikke navngitt',
+                },
+            })
+        } else {
+            dispatch({
+                type: 'addTile',
+                tile: {
+                    type: 'stop_place',
+                    placeId: selectedStopPlace.value,
+                    name:
+                        selectedStopPlace.label.split(',')[0] ??
+                        'Ikke navngitt',
+                },
+            })
+        }
+        setSelectedStopPlace(null)
+        setSelectedQuay(null)
+    }
+
+    const handleCreateBoard = async () => {
+        if (!board?.tiles?.length) {
+            addToast({
+                title: 'Ingen holdeplasser er lagt til',
+                content: 'Vennligst legg til holdeplasser',
+                variant: 'info',
+            })
+            return
+        }
+        const response = await createBoardRequest(
+            board?.tiles ?? [],
+            board?.meta?.title ?? '',
+        )
+        router.push(`/edit/${response.bid}`)
+        router.reload()
+    }
+
+    return (
+        <div className="flexColumn h-100">
+            <Heading3>Legg til holdeplasser i Tavla </Heading3>
+            <Paragraph>
+                Logg inn for å få tilgang til å opprette og administrere tavler.
```

Comment:
Kanskje litt lang setning?
Mener vi kan forenkle til `Logg inn for å opprette og administrere tavler.`

Reply:
Endret teksten til det som står i skissen til Maiken i figma🚂